src/main/java/org/opentripplanner/api/model/Itinerary.java

```diff
@@ -54,6 +55,8 @@
 
     public String itineraryType;
 
+    public BigDecimal price;
```

Comment:
Nie wiem czy to było ustalane z Radkiem, może warto zwracać dodatkowo koszt każdego Stepa. Ale to szczegół

Reply:
Nie wiem, czy w ogóle da się zwracać koszt każdego stepu, skoro istnieje coś takiego jak bilet przesiadkowy :P

Reply:
True :D